services/galley/test/integration/API/MLS.hs

```diff
@@ -752,6 +743,7 @@ testRemoteAppMessage = do
           "on-conversation-updated" -> pure (Aeson.encode ())
           "on-new-remote-conversation" -> pure (Aeson.encode EmptyResponse)
           "on-mls-message-sent" -> pure (Aeson.encode EmptyResponse)
+          "mls-welcome" -> pure (Aeson.encode EmptyResponse)
```

Comment:
Why is this needed in existing test cases? I'd assume the endpoint isn't called, otherwise the catch-all clause would trigger and we'd get an error.

That said, is is needed in the tests introduced by this PR?

Reply:
Errors get thrown actually when it's not implemented, however the tests still pass.